src/studio-header/UserMenu.jsx

```diff
@@ -17,22 +19,24 @@ const UserMenu = ({
   // injected
   intl,
 }) => {
+  const showUsername = !getConfig().ENABLE_HEADER_WITHOUT_USERNAME;
+  const avatarAlt = showUsername ? username : name;
```

Comment:
can we update the code by keeping the same logic as the feature name suggests and not negate the value?

```suggestion
  const hideUsername = !getConfig().ENABLE_HEADER_WITHOUT_USERNAME;
  const avatarAlt = hideUsername ? name : username;
```